docs/modules/ROOT/pages/how-to-guides/proc_maintaining_operator_graphs.adoc

```diff
@@ -0,0 +1,174 @@
+
+
+For additional information, see link:https://olm.operatorframework.io/docs/reference/file-based-catalogs/[File based Catalog].
+
+== Setting up an FBC Compatible git repository
+
+An FBC-compatible git repository has:
+
+* A Docker file at the root of the FBC repository. This file has the `ose-operator-registry` binary images. 
+A `yaml` file (for example, `catalog-config.yml`) that targets different Red Hat OpenShift Container Platform (OCP) versions. This `yaml` file contains important metadata of the operator. 
+
++
+*Example:*
+
++
+[source,yaml]
+----
+schema: olm.composite.catalogs
+catalogs:
+  - name: "4.13" <1>
+    destination:
+      baseImage: registry.redhat.io/openshift4/ose-operator-registry:v4.13 <2>
+      workingDir: .
+    builders:
+      - olm.builder.basic
+  - name: "4.12"
+    destination:
+      baseImage: registry.redhat.io/openshift4/ose-operator-registry:v4.12 This template can be hosted
+      workingDir: .
+    builders:
+      - olm.builder.basic
+----
+<1> The OCP version.
+<2> The binary image that the system uses to build the FBC fragments.
+
+
+* For every targeted OCP version, you should have a directory with a `devfile` that {ProductName} uses to detect the type of component. It also contains information about how to set up a build. 
+
++	
+*Example:*
+
++
+[source,yaml]
+----
+schemaVersion: 2.2.0
+metadata:
+  name: fbc-4.12
+  displayName: FBC 4.12
+  description: 'File based catalog'
+  language: fbc <1>
+  provider: Red Hat
+components:
+  - name: image-build
+    image:
+      imageName: ""
+      dockerfile:
+        uri: catalog.Dockerfile
+        buildContext: "4.12"
+  - name: kubernetes
+    kubernetes:
+      inlined: placeholder
+    attributes:
+      deployment/container-port: 50051
+      deployment/cpuRequest: "100m"
+      deployment/memoryRequest: 512Mi
+      deployment/replicas: 1
+      deployment/storageRequest: "0"
+commands:
+  - id: build-image
+    apply:
+      component: image-build"
+----
+<1>  Indicates the component that {ProductName} builds.
+
+== Adding a component
+Users should manage all the FBC components through a single application. This centralized management enables synchronized updates for graphs across all OpenShift versions.
+
+.Prerequisites
+
+* You have access to link:https://console.redhat.com/beta/hac/application-pipeline[App Studio].
+* You have FBC compatible git repository.
+* You have to link:https://redhat-appstudio.github.io/docs.appstudio.io/Documentation/main/how-to-guides/Import-code/proc_importing_code/#creating-a-red-hat-container-registry-token[create a Red Hat Container Registry token].
+
+.Procedures
+
+. Go to App Studio and select Create application.
+. In the *Git repository URL* field, enter the URL of your FBC-compatible Git repository.
+. Select *Import code*. The system displays the Configure your components for deployment page.
```

Comment:
Should we prompt the user to deselect all components which are _not_ FBC or that they don't want to onboard? Or to ensure that all desired components are present?

Reply:
It is possible to import the same repo multiple times. RHTAP just changes the automatic name, it doesn't de-select that component. The UI can be used to de-select detected components, and as long as one is selected, they can import.

Reply:
Thank you, Andrew and Matt
As per the conversation, I think we do not need any updates here.